GoogleForCreators/web-stories-wp
# Added delayed tooltip

## Context

This adds a delayed tooltip to the library tabs.

* Delay before showing: 1 second
* Delay before requiring fresh delay again to show: 0.5 second

## User-facing changes

_Demo of delayed tooltip:_
![delayed-tooltip](https://user-images.githubusercontent.com/637548/115782967-48f8a080-a38a-11eb-93d2-fcbd255815bf.gif)

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Hover over the tabs in the library
2. Observe a delayed tooltip
3. Observe that if moving from one tab to another quickly (less than 500ms off tab) once the tooltip is displayed, the tooltip appears immediately over the next tab.

### UAT




## Checklist




---

Fixes #7057
